pavelib/assets.py

```diff
@@ -763,9 +763,9 @@ def webpack(options):
     Run a Webpack build.
     """
     settings = getattr(options, 'settings', Env.DEVSTACK_SETTINGS)
-    static_root_lms = Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
+    result = Env.get_django_setting("STATIC_ROOT,WEBPACK_CONFIG_PATH", "lms", settings=settings)
+    static_root_lms, config_path = result.encode('utf-8').splitlines()
```

Comment:
why we need to encode string to `utf-8`?

Reply:
On `edly-io` repo, `get_django_setting` was returning `unicode` encoded value, but it was updated in `edx/master`. I checked it after the comment so there is no need for `utf-8` anymore, updated the code.